private/pkg/transport/http/httpserver/runner.go

```diff
@@ -96,7 +96,17 @@ func (s *runner) Run(
 			return err
 		}
 	}
+	return s.serve(ctx, listener, mux)
+}
 
+// serve is intended to only be called within the httpserver package by
+// the Run function. Once callers have migrated away from NewRunner().Run()
+// it would be ideal to move this logic into the Run function.
+func (s *runner) serve(
+	ctx context.Context,
+	listener net.Listener,
+	mux http.Handler,
```

Comment:
Maybe pick one for consistency to describe an http.Handler? It's called `handler http.Handler` in the Run function